packages/core-data/src/entity-provider.js

```diff
@@ -74,16 +74,17 @@ export function useEntityId( kind, type ) {
  * specified property of the nearest provided
  * entity of the specified type.
  *
- * @param {string} kind The entity kind.
- * @param {string} type The entity type.
- * @param {string} prop The property name.
+ * @param {string} kind  The entity kind.
+ * @param {string} type  The entity type.
+ * @param {string} prop  The property name.
+ * @param {string} [_id] A fallback entity ID to use if no entity provider exists.
```

Comment:
```suggestion
 * @param {string} [_id] An entity ID to use instead of the context-provided one.
```

The fallback should be the provider, otherwise, we would have no way of overwriting root contexts.

Reply:
It makes things a little trickier this way because we can't conditionally call hooks

Reply:
Not if the ID parameter doesn't change for the lifetime of the component, which it shouldn't.